### PR TITLE
Add monochrome icon for android 13

### PR DESCRIPTION
**Pull request by Donnnno**
A monochrome icon for android 13

### CODE REVIEW

File: platform/android/java/lib/res/mipmap-anydpi-v26/icon.xml [hunk 1]
```diff
@@ -2,4 +2,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@mipmap/icon_background"/>
     <foreground android:drawable="@mipmap/icon_foreground"/>
+	<monochrome android:drawable="@drawable/icon_monochrome"/>
```
Comment (YuriSizov) [inline platform/android/java/lib/res/mipmap-anydpi-v26/icon.xml:5]: Should be spaces instead of tabs.

**Review by m4gr3d — APPROVED** [review]
Can you provide a screenshot for what the monochromatic icon looks like on Android 13.

**Comment by akien-mga** [comment]
I rebased and squashed the commits myself.

We still need to see how the icon looks like before merging though. Seems to be some pseudo-SVG but Inkscape can't read it, if someone has Android Studio installed and can take screenshots, that would be welcome.

I don't have any Android 13 device to test it either.

**Comment by m4gr3d** [comment]
The icon is too big and doesn't look right in the app library

godot_screenshot_icon_monochrome
godot_icon_monochrome

**Review by m4gr3d — CHANGES_REQUESTED** [review]
The icon needs to be resized to match the guidelines at https://developer.android.com/develop/ui/views/launch/icon_design_adaptive#design-adaptive-icons

**Comment by akien-mga** [comment]
Closing as this seems to be abandoned by the original author.

CC @MewPurPur @Calinou if you're interested.

For the record the "icon too big" issue seems to be true for the default, colored icon too, and would be good to address one way or another (either adding a new default icon for Android that matches the adaptive icon guidelines, or adding export code that takes care of properly padding it, as I see also a lot of Godot games on Android having way too zoomed in icons).

**Pull request closed without merging**